### ISSUE TITLE
how to pretrain?

### ISSUE THREAD

**Issue by fangxu622** [Media-Smart/vedaseg#5]
if I want to train models use my own VOC dataset base on existing models. 

modify this?
resume = "./deeplabv3_resnet101_voc_epoch_50.pth"?

**Comment by hxcai** [comment]
@fangxu622 Hi~For example, you can use resume=dict(checkpoint=your_path, resume_optimizer=False, resume_lr=False, resume_epoch=False), it means loading pretrained model weights, and not resuming optimizer, learning rate and epoch.
